mkt/reviewers/views.py

```diff
@@ -277,7 +277,7 @@ def _review(request, addon, version):
     all_forms = [form, attachment_formset, testedon_formset]
 
     if version:
-        features_list = [unicode(f) for f in version.features.to_list()]
+        features_list = [unicode(f) for f in version.features.to_names()]
```

Comment:
the names are lazily translated, so if you don't do this, you get proxy objects instead of unicode strings.